_articles/getting-paid.md

```diff
@@ -122,6 +122,14 @@ Organizational funding might go towards paying current contributors, covering th
 
 As open source's popularity increases, finding funding for projects is still experimental, but there are a few common options available.
 
+There are many good funding places and options are available such as
```

Comment:
I don't think this list works right here in the article. Read the previous sentence, it is very similar, and the following sections flow from it.

Maybe some of the links belong in the following sections? Or some reorganization of the following sections?

Reply:
:/ What do you think should be fixed?

Correct heading

> Finding funding for your project

and then list the options, no?

Reply:
It looks like they are all grants. They should be worked into the existing apply for grants section below. In other words, I suggest improving that section rather than adding a new list.

Reply:
I added a line and few improvements, let me know if ok.